tools/dpdk_ptf.Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ghcr.io/ipdk-io/ipdk-ubuntu2004-x86_64:ipdk_v23.07
```

Comment:
`tools` might be a strange folder for this file. Maybe create a `docker` folder?